Scripts/DCS-BIOS/lib/modules/Module.lua

```diff
@@ -977,6 +977,33 @@ function Module:defineEjectionHandleSwitch(identifier, device_id, command, arg_n
 	return control
 end
 
+--- Adds a new string output containing the frequency of the device
+--- @param identifier string the unique identifier for the control
+--- @param device_id integer the dcs device id
+--- @param max_length integer the maximum length of the string
+--- @param category string the category in which the control should appear
+--- @param description string additional information about the control
+--- @return Control control the control which was added to the module
+function Module:defineFrequencyString(identifier, device_id, max_length, category, description)
+	local alloc = self:allocateString(max_length, identifier)
+	self:addExportHook(function(dev0)
+		local value = tostring(GetDevice(device_id):get_frequency())
+		if value == nil then
```

Comment:
I don't believe tostring can return nil - only "nil" (as a string). If get_frequency can return nil, we should make sure it's documented as such and perform the check on that result.